galata/test/jupyterlab/terminal.test.ts

```diff
@@ -103,3 +103,26 @@ test('Terminal should open in Launcher cwd', async ({ page, tmpPath }) => {
   await page.waitForTimeout(1000);
   expect(await terminal.screenshot()).toMatchSnapshot('launcher-term.png');
 });
+
+test('Terminal web link', async ({ page, tmpPath }) => {
+  await page.waitForSelector(`.jp-Launcher-cwd > h3:has-text("${tmpPath}")`);
+
+  await page.locator('[role="main"] >> p:has-text("Terminal")').click();
+
+  const terminal = page.locator(TERMINAL_SELECTOR);
+  await terminal.waitFor();
+
+  await page.waitForTimeout(1000);
+  await page.keyboard.type('echo https://jupyter.org/');
+  await page.keyboard.press('Enter');
+  await page
+    .locator('[role="region"] canvas')
+    .nth(3)
+    .hover({
+      position: {
+        x: 100,
+        y: 20
```

Comment:
Yes, this will be flaky depending on the font which will vary between operating systems. Let's try:

```suggestion
        x: 80,
        y: 33
```